src/game/start.py

```diff
@@ -58,43 +58,43 @@ def create_grid_string(grid):
     return display_string
 
 
-def create_pieces_string(remaining_pieces, game_turn):
+def pieces_to_string(remaining_pieces, game_turn):
     display_string = 'Remaining pieces :\n'
     i = 1
     for piece in remaining_pieces:
         display_string += ' '
-        display_string += select_piece_string(piece, game_turn)
+        display_string += selected_piece_to_string(piece, game_turn)
         display_string += ' '
     return display_string
 
 
-def select_piece_string(piece_number, game_turn):
+def selected_piece_to_string(piece_number, game_turn):
     if game_turn.selected_piece == piece_number:
         return "[" + str(piece_number) + "]"
     else:
         return str(piece_number)
 
 
-def create_players_string(game_turn):
+def players_to_string(game_turn):
     display_string = ''
     player_1 = 'Player 1'
     player_2 = 'Player 2'
-    player_1 = select_player_string(player_1, game_turn.player_one_active)
-    player_2 = select_player_string(player_2, not game_turn.player_one_active)
+    player_1 = selected_player_to_string(player_1, game_turn.player_one_active)
```

Comment:
You can directly set 'Player N' string in selected_player_string first arg to avoid unecessary memory allocation